client/src/components/PizzaPlaceShow.js

```diff
@@ -39,6 +42,15 @@ const PizzaPlaceShow = props => {
 		)
 	}) 
 
+	let newReview = ""
```

Comment:
we could opt for `newReview` to initially be `undefined`:
```js
let newReview;
```
 rather than a string, since it is either going to be nothing (undefined if there is no currentUser) or will be an actual element when there is a currentUser